docs-src/real_time_messaging.rst

```diff
@@ -80,40 +83,14 @@ Below is a code snippet that uses the legacy version of ``RTMClient``. For new a
 
     slack_token = os.environ["SLACK_BOT_TOKEN"]
     rtm_client = RTMClient(token=slack_token)
-    rtm_client.start()
+    rtm_client.connect()
 
 **rtm.start vs rtm.connect (v1 client)**
 
-By default, the RTM client uses ``rtm.connect`` to establish a WebSocket connection with Slack. The response contains basic information about the team and WebSocket url.
-
-If you'd rather use ``rtm.start`` to establish the connection, which provides more information about the conversations and users on the team, you can set the ``connect_method`` option to ``rtm.start`` when instantiating the RTM Client. Note that on larger teams, use of ``rtm.start`` can be slow and unreliable.
-
-.. code-block:: python
-
-    import os
-    from slack_sdk.rtm import RTMClient
-
-    @RTMClient.run_on(event="message")
-    def say_hello(**payload):
-        data = payload['data']
-        web_client = payload['web_client']
-        if 'text' in data and 'Hello' in data['text']:
-            channel_id = data['channel']
-            thread_ts = data['ts']
-            user = data['user'] # This is not username but user ID (the format is either U*** or W***)
-
-            web_client.chat_postMessage(
-                channel=channel_id,
-                text=f"Hi <@{user}>!",
-                thread_ts=thread_ts
-            )
+.. parsed-literal ::
+   **rtm.start method deprecation after Nov 30th, 2021.** see details `here <https://api.slack.com/changelog/2021-10-rtm-start-to-stop>`_
```

Comment:
```suggestion
   **rtm.start method has been deprecated for apps created after Nov 30th, 2021.** see details `here <https://api.slack.com/changelog/2021-10-rtm-start-to-stop>`_
```

Reply:
isn't it fully deprecated?

Reply:
The announcement has this caveat: 

> For apps created before November 30, 2021, making API calls to https://slack.com/api/rtm.start will still technically succeed and return successful, but highly truncated responses even after September 27, 2022.

Seems like newly created apps after the date must use rtm.connect, and older apps will receive responses from rtm.start that are identical to rtm.connect. Is that your reading as well?

Reply:
Ahh i see Thank you!
I signed the CLA